nca/Resources/IngressPolicy.py

```diff
@@ -39,8 +40,8 @@ def contained_in(self, other):
 class IngressPolicy(NetworkPolicy):
     """
     This class implements ingress controller logic for incoming http(s) requests
-    The logic is kept similarly to NetworkPolicy, where the selected_peers are the ingress controller peers,
-    and the rules are egress_rules.
+    The logic is kept similarly to NetworkPolicy, where the selected_peers are the ingress/egress controller peers,
+    and the rules are ingress/egress_rules.
```

Comment:
Should we consider renaming this class? `IngressPolicy` is no longer for ingress gateway, but also for egress gateway.

Reply:
I see it used also to represent k8s Ingress policy, so maybe remove the `Istio` from the name?
Also, documentation of this class should specify the relevant resources (Istio Ingress/Egress gateway and k8s Ingress )